.github/workflows/release.yml

```diff
@@ -0,0 +1,76 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  main:
+    name: Create Protobuf generated files
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup GO
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.17.1
+      - run: npm install -g typescript
+
+      - name: Setup protoc-gen-go
+        uses: arduino/setup-protoc@v1
+
+      - name: Add Protobuf files
+        run: |
+          go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.27.1
+          go mod download
+          make generate
+
+      - name: Add proto and pb files to release
+        run: |
+          zip generated_proto_files.zip ./fig -r
+          zip generated_pb_go.zip ./pb -r
+
+      - name: Setup Graph Proto as Schema
+        run: |
+          git clone https://github.com/figment-networks/graph-proto-as-schema
+          cd graph-proto-as-schema
+          yarn install
+          tsc
+          mkdir outp
+          node ./dist/index.js -o ./outp  -d ../fig/tendermint/codec/v1
+
+      - name: Add ts files to release
+        run: |
+          zip generated_ts_files.zip ./graph-proto-as-schema/outp -r
+
+      - name: Graph-ts
+        run: |
+          git clone --branch cosmos --single-branch https://github.com/figment-networks/graph-ts
+          cp -r ./graph-proto-as-schema/outp ./graph-ts/chain
```

Comment:
Have you tried running `graph-as-to-rust` on the file generated by `graph-proto-as-schema` (after putting it under `graph-ts/chain`)? Maybe we don't need the whole `graph-ts` repository.

Reply:
yeah it needs some of the other files (collections etc). this is the quickest way of doing it